draft-ietf-jose-json-proof-algorithms.md

```diff
@@ -679,42 +679,34 @@ The final proof value from the Issuer is an array with the octets of the header
 <{{./fixtures/template/jpt-issuer-payloads.json}}
 Figure: Issuer payloads (JSON, as array)
 
-The resulting JSON serialized JPT using the above examples is:
-
-<{{./fixtures/build/su-es256-issuer.json.jwp.wrapped}}
-Figure: Issued JWP (SU-ES256, JSON Serialization)
-
 The compact serialization of the same JPT is:
 
 <{{./fixtures/build/su-es256-issuer.compact.jwp.wrapped}}
-Figure: Issued JWP (SU-ES256, Compact Serialization)
+Figure: Issued JWP (SU-ES256, JSON, Compact Serialization)
```

Comment:
Do we want to use the language "JSON Compact Serialization" anymore or just call it the "JSON Serialization"?